apps/balancer-tools/src/app/apr/(utils)/rewardsApr.ts

```diff
@@ -0,0 +1,133 @@
+import { PoolRewardsQuery } from "@bleu-balancer-tools/gql/src/balancer-rewards/__generated__/Ethereum";
+import { NetworkChainId } from "@bleu-balancer-tools/utils";
+
+import { rewards } from "#/lib/gql/server";
+import { ArrElement, GetDeepProp } from "#/utils/getTypes";
+
+import { fetchPoolAveragesForDateRange } from "./calculatePoolStats";
+
+type RewardInfo = ArrElement<
+  GetDeepProp<PoolRewardsQuery, "rewardTokenDeposits">
+>;
+
+export async function getRewardsAprForDateRange(
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  poolId: string,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  network: string,
+  startAt: number,
+  endAt: number,
+) {
+  const tempPoolId =
+    "0x17e7d59bb209a3215ccc25fffef7161498b7c10d000200000000000000000020";
```

Comment:
It doesn't break the app, but yes, it will return the same results as said in the video. I did just like we did for yield APR. At first, we used rETH everywhere to make things run to then generalize for every pool/network. In this case, we are always getting the return from

https://app.balancer.fi/#/base/pool/0x17e7d59bb209a3215ccc25fffef7161498b7c10d000200000000000000000020